### PR TITLE
Create spanish translations

### PR DESCRIPTION
**Pull request by Ismael034**
Hi,
I have created translaions to spanish to Photok

Thanks

### CODE REVIEW

**Comment by Ismael034** [comment]
It could be great if you add 'hacktoberfest' as a topic

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    <!-- DELETE -->
+    <string name="delete_are_you_sure">¿Estas seguro de que quieres <b>borrar</b> %1$d foto(s)?</string>
+    <string name="delete_are_you_sure_this">¿Estas seguro de que quieres <b>borrar</b> esta foto?</string>
+    <string name="delete_deleting">Borrando foto(s)…</string>
+
+    <!-- EXPORT -->
+    <string name="export_are_you_sure">¿Estás seguro de que quieres <b>exportar</b> %1$d foto(s), en tu galería?</string>
+    <string name="export_are_you_sure_this">Are you sure, you want to <b>export</b> this photo, back to your gallery?</string>
+    <string name="export_exporting">Exporting photo(s)…</string>
+    <string name="export_permission_rationale">Photok needs to access your storage, to export photos.</string>
```
Comment (leonlatsch) [inline app/src/main/res/values-es/strings.xml:95]: Missing translations

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    <string name="unlock_wrong_password">\u26A0 Contraseña incorrecta</string>
+
+    <!-- GALLERY -->
+    <string name="gallery_placeholder">No hay fotos en tu baúl</string>
+
+    <!-- PROCESS -->
+    <string name="process_close">Cerrar</string>
+    <string name="process_abort">Cancelar</string>
+    <string name="process_finished">Terminado</string>
+    <string name="process_initialize">Preparando...</string>
```
Comment (leonlatsch) [inline app/src/main/res/values-es/strings.xml:64]: should be unicode: "…"

**Review by leonlatsch — CHANGES_REQUESTED** [review]

**Comment by Ismael034** [comment]
Sorry, I have fixed the issues. 

Thanks

**Comment by leonlatsch** [comment]
@Ismael034 Are you willing to continue translating photok? Otherwise, this is just a "bad handled Task" for me to do.

**Review by leonlatsch — APPROVED** [review]

**Comment by Ismael034** [comment]
Yes, I will! I think it is a great app I want to support it

**Comment by leonlatsch** [comment]
Okay, otherwise, I would have to remove it again before releasing 1.0.0,
Thank you for your contribution

**Pull request merged**